mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenPropertyResolver.java

```diff
@@ -18,4 +23,13 @@ static private boolean isSkip() {
         return Boolean.valueOf(System.getProperty("maven.test.skip", "false"));
     }
 
+    static boolean isSpecificTestClassSet() {
+        String specificTestClasses = System.getProperty("test");
+        return specificTestClasses != null && !containsPattern(specificTestClasses);
+    }
+
+    private static boolean containsPattern(String specificTestClasses) {
+        Matcher matcher = TEST_CLASS_PATTERN.matcher(specificTestClasses);
```

Comment:
this doesn't match the "two tests" case that I mentioned above:
`-Dtest=TestOne,TestTwo`
IIRC our discussion, in this case, it should run both of them & disable ST - unless there is a regex inside of any test name:
`-Dtest=TestOne,TestT*`
then the ST should be activated.
Am I correct?